contracts/Comet.sol

```diff
@@ -1262,14 +1262,15 @@ contract Comet is CometMainInterface {
      * @dev Only callable by governor
      * @dev Note: Setting the `asset` as Comet's address will allow the manager
      * to withdraw from Comet's Comet balance
+     * @dev Note: For USDT, if there is non-zero prior allowance, it must be reset to 0 first before setting a new value in proposal
      * @param asset The asset that the manager will gain approval of
      * @param manager The account which will be allowed or disallowed
      * @param amount The amount of an asset to approve
      */
     function approveThis(address manager, address asset, uint amount) override external {
         if (msg.sender != governor) revert Unauthorized();
 
-        IERC20NonStandard(asset).approve(manager, amount);
+        ERC20(asset).approve(manager, amount);
```

Comment:
Do we have tests for this? I feel this would revert right since we're not using a low-level call right?

Reply:
Looks like scenarios are not running properly in CI, but this test should probably fail: `Comet#approveThis > allows governor to authorize and rescind authorization for non-Comet ERC20`

Reply:
updated to use low-level call as well. And checked on success and bool return.